sklearn/utils/tests/test_utils.py

```diff
@@ -186,3 +186,36 @@ def to_tuple(A):    # to make the inner arrays hashable
     S = set(to_tuple(A))
     shuffle(A)  # shouldn't raise a ValueError for dim = 3
     assert_equal(set(to_tuple(A)), S)
+
+
+def test_shuffle_pass_through():
```

Comment:
shuffle_dont_convert_indexable? shuffle_pass_through_indexable? shuffle_dont_convert_to_array? (I think the last)